src/Skills.jsx

```diff
@@ -91,6 +92,14 @@ class Skills extends Component {
      });
   }
 
+  reloadSkills() {
```

Comment:
If this has the same functionality as componentDidMount(), you should have that method call this too.  No reason to call the same axios.get() in 2 different places